torch/_inductor/codegen/cpp.py

```diff
@@ -3857,8 +3856,8 @@ def get_call_ranges(node: BaseSchedulerNode):
                 call_ranges = tuple(group) + tuple(reduction_group)
                 return call_ranges
 
-            LocalBuffer = namedtuple("LocalBuffer", ["local_buf", "global_buf"])
-            local_buffers: List[LocalBuffer] = []
+            BufferMapper = namedtuple("BufferMapper", ["local_buf", "global_buf"])
```

Comment:
It is 1:n mapping from local buffer to global ones. Is it better to make it a dict mapping local buffer to a list of global buffers?

Reply:
Thanks for the suggestion. Added a list of buffers to record all the local_buffers and a dict mapping each local buffer name to a list of global buffers.